deps/src/polymake.cpp

```diff
@@ -75,10 +70,7 @@ JLCXX_MODULE define_module_polymake(jlcxx::Module& polymake)
         bool full=false, bool html=false){
         std::vector<std::string> ctx_help =
             data.main_polymake_session->shell_context_help(input, pos, full, html);
-        jl_value_t** doc_strings = new jl_value_t*[ctx_help.size()];
-        for (int i=0; i < ctx_help.size(); ++i)
-            doc_strings[i] = jl_cstr_to_string(ctx_help[i].c_str());
-        return jlcxx::make_julia_array(doc_strings, ctx_help.size());
+        return jlcxx::make_julia_array(&ctx_help[0], ctx_help.size());
```

Comment:
Why is it no more necessary to convert from c-string to string?

Reply:
I don't know why this was necessary in the first place;
Example here: https://github.com/JuliaInterop/libcxxwrap-julia/commit/b0068f102b190ece10bdfc9f7ac096c4d2ceefd0
shows that this is not needed